.buildkite/pipeline.yml

```diff
@@ -37,8 +32,17 @@ steps:
       bundle install
       bundle exec fastlane test_without_building name:WordPressUnitTests try_count:3
     depends_on: "build"
+
+  #################
+  # Lint Translations
+  #################
+  - label: "🧹 Lint Translations"
+    command: "gplint /workdir/WordPress/Resources/AppStoreStrings.po"
+    plugins:
+      - docker#v3.8.0:
+          image: "jkmassel/glotpress-linter:latest"
```

Comment:
What's the difference between using the [docker plugin from BuildKite](https://github.com/buildkite-plugins/docker-buildkite-plugin) and declaring an `agents:` key for this step and use a Docker agent 🤔

Reply:
@jkmassel btw, now that you apparently transferred this repo to `wordpress-mobile/glotpress-linter` a couple of days ago, we should update this line 😉

Reply:
> What's the difference between using the docker plugin from BuildKite and declaring an agents: key for this step and use a Docker agent

The agents key just causes this step to run on a machine that contains docker. The docker plugin handles pulling the image down and running our commands inside of the container. We could accomplish this ourselves too, but given the fact that this is a first-party plugin it doesn't seem worth the effort IMHO? 🤷

Reply:
> btw, now that you apparently transferred this repo to wordpress-mobile/glotpress-linter a couple of days ago, we should update this line 😉

Yes indeed – just need to figure out the Docker Hub stuff for this – I'm not sure we'll be able to continue using auto build, and we'll want to host these images in AWS anyway. For now this works and gives us the protection of the change, so I figured landing it soon was more important. It wasn't moved, rather copied, so it's still available at https://hub.docker.com/r/jkmassel/glotpress-linter. 

But yes, this will definitely need to be updated once we have a home for our images!

Reply:
@AliSoftware may have made the same "mistake" as I have in assuming that `jkmassel/glotpress-linter` is the Github project name 🙈

Reply:
Addressed this in #16847

Reply:
💙